pykafka/utils/compression.py

```diff
@@ -29,11 +29,32 @@
 except ImportError:
     snappy = None
 
+try:
+    import lz4.frame as lz4
+except ImportError:
+    lz4 = None
+
+#try:
+#    import lz4f
```

Comment:
Can you explain what's going on here? Why might we need `lz4f` instead of `lz4`?

Reply:
Looking deeper into [kafka-python codec.py](https://github.com/dpkp/kafka-python/blob/a00f9ead161e8b05ac953b460950e42fa0e0b7d6/kafka/codec.py#L205-L216) it looks like it's there to support interoperable lz4 framing.

See [KIP-57](http://grokbase.com/t/kafka/dev/164sdz8cvb/kip-57-interoperable-lz4-framing)

Reply:
What's your intention with having it commented here? Is it a placeholder? I don't see it being used anywhere other than the check for the presence of an `lz4` library.